app/routes/app/s/$id.tsx

```diff
@@ -0,0 +1,143 @@
+import { Search16 } from "@carbon/icons-react";
+import { Detail } from "~/components/Detail";
+import * as Author from "~/components/Author";
+import { Badge, Button, Center, Divider, Title, Text, Avatar, Input, Select, Checkbox, Paper } from "@mantine/core";
+import { Form, Link } from "@remix-run/react";
+
+export default function ChallengeSubmission() {
+  const isWinner = true;
+
+  const reviews = [
+    { id: "1", author: "1234" },
+    { id: "2", author: "2234" },
+    { id: "3", author: "3234" },
+  ];
+
+  return (
+    <div className="mx-auto container mb-12 px-10">
+      <section className="flex flex-wrap gap-5 justify-between pt-12 pb-10">
+        <div className="flex items-center gap-2">
+          <Title order={2}>Question Title </Title>
+          {isWinner ? <Avatar size="sm" src="/img/trophy.svg" /> : <></>}
+        </div>
+        <Center className="flex flex-wrap gap-5">
+          <Link to="/app/t/[topicId]/submission/review">
+            <Button radius="md" className="mx-auto">
+              Review Question
+            </Button>
+          </Link>
+        </Center>
+      </section>
+      <section className="flex flex-col space-y-7 pb-24">
+        <div className="flex flex-wrap gap-x-8">
```

Comment:
Adding `gap-y-4` will make this look 👌 on mobile (actually didn't know you could use gap with flex 😶‍🌫️ )